src/stun.c

```diff
@@ -508,6 +508,39 @@ bool is_stun_datagram(const void *data, size_t size) {
 	return true;
 }
 
+bool is_stun_datagram2(const void *data, size_t size) {
```

Comment:
What is the difference with `is_stun_datagram`?

Reply:
> 
> 
> What is the difference with `is_stun_datagram`?

TCP a stream-oriented transmission protocol, more than one packet may be received in one recv() call.

STUN Packet + STUN Packet + ...
STUN Packet + Non STUN Packet + ...
...